apps/barista-design-system/src/components/live-example/live-example.ts

```diff
@@ -237,19 +227,20 @@ export class BaLiveExample implements OnDestroy {
   }
 
   private _initExample(): void {
-    const exampleType = EXAMPLES_MAP.get(this._name);
-    if (exampleType) {
-      const factory = this._componentFactoryResolver.resolveComponentFactory(
-        exampleType,
-      );
-      this._componentRef = createComponent(
-        factory,
-        this._viewContainerRef,
-        this._injector,
-        this._placeholder.nativeElement,
-        true,
-      );
-    }
+    this.example$ = from(
+      import(`../../../../../libs/examples/src/${this.directory}/index.ts`),
```

Comment:
All the dots. We have too many folders :D

Reply:
we have to use this because otherwise, webpack cannot detect the files with the static analysis.

If we put this into a variable it would break. Webpack is doing a static analysis of all the folders that are detected with the es6 imports and creating chunks for it.